deploy/ocp-templates/glusterfs-template.yaml

```diff
@@ -37,6 +37,10 @@ objects:
           imagePullPolicy: IfNotPresent
           name: glusterfs
           env:
+          # set GLUSTER_BLOCKD_STATUS_PROBE_ENABLE to "1" so the
+          # readiness/liveness probe validate gluster-blockd as well
+          - name: GLUSTER_BLOCKD_STATUS_PROBE_ENABLE
+            value: "1"
```

Comment:
All parameters are defined below - but missing this variable.

Reply:
Also, the default value in the parameters section. (check other variables for reference)